proposals/csharp-10.0/GlobalUsingDirective.md

```diff
@@ -1,5 +1,7 @@
 # Global Using Directive
 
+[!INCLUDE[Specletdisclaimer](~/docs/csharp/includes/speclet-disclaimer.md.md)]
```

Comment:
I can't say that I'm a fan of this, as while it makes the experience better on docs.microsoft.com, it makes it worse here: this link does not resolve, and it appears at the top of every speclet.

Reply:
I could move the include file into this repo, then the link will resolve. That would help somewhat. (It would also move the issue, because the include file has links to other articles on docs.

I'm ambivalent about this as well. But, it does provide clarity for those customers reading the speclets and finding that they may include designs that were modified by later LDM decisions.